Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN npm install
 FROM install as build
 WORKDIR /app
 COPY . /app
+RUN cp ./src/SideBarPathApp.tsx ./src/components/DashboardNavbar/SideBarPath.tsx 
```

Comment:
Wtf?

Reply:
two of files have the same content as you have seen but I don't want to import that because some reason